tests/operators/test_agnostic_boolean_check.py

```diff
@@ -53,194 +53,116 @@ def drop_table_snowflake(
     snowflake_conn.close()
 
 
-class TestBooleanCheckOperator(unittest.TestCase):
-    """
-    Test Boolean Check Operator.
-    """
-
-    cwd = pathlib.Path(__file__).parent
-
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.table = "boolean_check_test"
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=Table(
-                cls.table,
-                conn_id="postgres_conn",
-                database="pagila",
-                schema="public",
-            ),
-        ).operator.execute({"run_id": "foo"})
-
-        cls.snowflake_table = get_table_name("boolean_check_test")
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=Table(
-                conn_id="snowflake_conn",
-                table_name=cls.snowflake_table,
-                schema=os.getenv("SNOWFLAKE_SCHEMA"),
-                database=os.getenv("SNOWFLAKE_DATABASE"),
-                warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
-            ),
-        ).operator.execute({"run_id": "foo"})
-
-        cls.table = "boolean_check_test"
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=Table(
-                cls.table,
-                conn_id="bigquery",
-                schema=SCHEMA,
-            ),
-        ).operator.execute({"run_id": "foo"})
-        cls.table_sqlite = Table(
-            "boolean_check_test", conn_id="sqlite_conn", schema="sqlite_schema"
-        )
-        aql.load_file(
-            path=str(cls.cwd) + "/../data/homes_append.csv",
-            output_table=cls.table_sqlite,
-        ).operator.execute({"run_id": "foo"})
-
-    @classmethod
-    def tearDownClass(cls):
-        drop_table_snowflake(
-            table_name=cls.snowflake_table,
-            schema=os.getenv("SNOWFLAKE_SCHEMA"),
-            database=os.getenv("SNOWFLAKE_DATABASE"),
-            warehouse=os.getenv("SNOWFLAKE_WAREHOUSE"),
-            conn_id="snowflake_conn",
-        )
-
-    def clear_run(self):
-        self.run = False
-
-    def setUp(self):
-        super().setUp()
-        self.clear_run()
-        self.addCleanup(self.clear_run)
-        self.dag = DAG(
-            "test_dag",
-            default_args={
-                "owner": "airflow",
-                "start_date": DEFAULT_DATE,
-            },
-        )
+@pytest.fixture(scope="module")
```

Comment:
This fixture is very similar to [test_agnostic_aggregate_check.py](https://github.com/astro-projects/astro/pull/181/files#diff-880cb6c270aa05815d7b53526ceb0f68dbc1a1f88d521b041a36cfefacbc7150)

Should we move it to `conftest.py` so we can re-use it and avoid code duplicates?

Reply:
This fixture closely resembles the tmp_table fixture, but without the upload capability. I want to make tmp_tables generic enough so that it can accept the file_path(s) as param and can also load a file into DB, should be simple enough.

Also, the default behavior should be able to just return tmp_table object without uploading a file to db, so it can be used in testcases like load_file where we just need a tmp_table object.